cpp/serve/request.h

```diff
@@ -53,6 +53,10 @@ class RequestNode : public Object {
    * top_p, repetition_penalty, max_gen_len, etc.
    */
   GenerationConfig generation_cfg;
+  /*!
+   * \brief The flag if request data is pinned in KVCache. Used for system prompt cache.
+   */
+  bool pinned = false;
```

Comment:
move this to generation config

Reply:
moved and added test for pinned system prompt.